### PR TITLE
astronomer: add page

### PR DESCRIPTION
**Pull request by Aracki**
I have added [astronomer](https://github.com/Ullaakut/astronomer) - a tool that scans repository for the stars from bot accounts.

### CODE REVIEW

File: pages/common/astronomer.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# astronomer
+
+> Tool that detects illegitimate stars from bot accounts on Github projects.
+> More information: <https://github.com/Ullaakut/astronomer>.
+
+- Scan repository:
```
Comment (mebeim) [inline pages/common/astronomer.md]:
```suggestion
- Scan a repository:
```

File: pages/common/astronomer.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`astronomer {{tldr-pages/tldr-node-client}}`
+
+- Scan maximum amount of stars in the repository:
```
Comment (mebeim) [inline pages/common/astronomer.md]:
```suggestion
- Scan the maximum amount of stars in the repository:
```

File: pages/common/astronomer.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`astronomer {{tldr-pages/tldr-node-client}} --stars {{50}}`
+
+- Scan repository including comparative reports:
```
Comment (mebeim) [inline pages/common/astronomer.md]:
```suggestion
- Scan a repository including comparative reports:
```

**Comment by sbrl** [comment]
Nice! I wonder what the result is for tldr-pages?

**Comment by mebeim** [comment]
I don't think I have te patience to find out... :joy: 
image

**Comment by schneiderl** [comment]
@mebeim any updates on it? 😄

**Comment by Aracki** [comment]
@mebeim I didn't set `tldr` as example on purpose, because of huge number of stars.

@sbrl I did it on another computer. I think it was **A**. I will share the screenshot tonight hopefully.

**Comment by mebeim** [comment]
@schneiderl it crashed at some point because GH API wasn't answering queries anymore :laughing: 

@Aracki makes sense!

**Comment by Aracki** [comment]
image

@sbrl @schneiderl @mebeim Here are the results.

**Pull request merged**